haystack/pipelines/utils.py

```diff
@@ -297,18 +305,22 @@ def _format_wrong_examples_report(
         "document_id", "context", "document_id_and_context", "document_id_or_context", "answer", "document_id_or_answer"
     ] = "document_id_or_answer",
     answer_scope: Literal["any", "context", "document_id", "document_id_and_context"] = "any",
+    wrong_examples_filter: List[str] = None,
+    max_characters_per_wrong_examples_report: int = None,
 ):
     examples = {
         node: eval_result.wrong_examples(
             node, document_scope=document_scope, answer_scope=answer_scope, n=n_wrong_examples
         )
         for node in eval_result.node_results.keys()
     }
-    examples_formatted = {
-        node: "\n".join(map(_format_wrong_example, examples)) for node, examples in examples.items() if any(examples)
-    }
+    examples_formatted = {}
+    for node, examples in examples.items():  # type: ignore
```

Comment:
Why do we need `type: ignore` here?

Reply:
mypy complained for some unintelligible reason :-) I'll revisit.